test/export/test_converter.py

```diff
@@ -25,7 +27,7 @@ def _check_equal_ts_ep_converter(self, mod, inp) -> ExportedProgram:
         # Check module.
         if isinstance(mod, torch.nn.Module):
             self.assertEqual(
-                ep.module().state_dict().keys(),
+                self._remove_lifted_tensor(ep.module().state_dict()).keys(),
```

Comment:
I think we should be comparing `ep.state_dict()` instead of `ep.module().state_dict()`. I updated here [https://github.com/pytorch/pytorch/pull/128442/files](https://github.com/pytorch/pytorch/pull/128442/files)